tests/test_setup.py

```diff
@@ -0,0 +1,13 @@
+import os
+from unittest.mock import patch, MagicMock
+
+os.environ["DATABASE_URL"] = "sqlite:///./test.db"
```

Comment:
to mock env vars you should use patch and fixture
https://docs.python.org/3/library/unittest.mock.html#patch-dict

Reply:
and any mutable object you want to patch you should use fixtures